docs/advanced_topics/customisation/custom_bulk_actions.rst

```diff
@@ -0,0 +1,123 @@
+Adding custom bulk actions
+==========================================
+
+This document describes how to add custom bulk actions to different listings
+
+
+Registering a custom bulk action
+--------------------------------
+
+    .. code-block:: python
+
+        from wagtail.admin.views.bulk_action import BulkAction
+        from wagtail.core import hooks
+
+
+        @hooks.register('register_bulk_action')
+        class CustomPageBulkAction(BulkAction):
```

Comment:
question: should this not inherit from `PageBulkAction` ? given this is the more generic example, I would call it `MyCustomBulkAction(BulkAction)`